tests/test/db-auth.js

```diff
@@ -20,7 +20,8 @@ describe('db-auth', function () {
 
   it('passes create with no comment', async function () {
     const bodyString = '{"label":"hello"}'
-    await db.create(agent, { bodyString })
+    const result = await db.create(agent, { bodyString })
+    expect(result.body).to.have.property('api:database_uri')
```

Comment:
Can't you check whether the URL is correct as well?

Reply:
It is!